app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -122,6 +123,18 @@ public void setVersion(Long version) {
 
     /** {@inheritDoc} */
     @Override
+    public boolean getExcludeStudyIdInExport() {
+        return excludeStudyIdInExport;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void setExcludeStudyIdInExport(boolean excludeStudyIdInExport) {
+        this.excludeStudyIdInExport = excludeStudyIdInExport;
+    }
```

Comment:
isExcludeStudyId vs isStudyIdExcludedInExport, latter is a little clearer to me

Reply:
Renamed